app/src/main/java/mozilla/lockbox/store/FingerprintStore.kt

```diff
@@ -84,7 +84,7 @@ open class FingerprintStore(
         get() = isFingerprintAuthAvailable || isKeyguardDeviceSecure
 
     open val isFingerprintAuthAvailable: Boolean
-        get() = fingerprintManager.isHardwareDetected && fingerprintManager.hasEnrolledFingerprints()
+        get() = (fingerprintManager?.isHardwareDetected ?: false) && (fingerprintManager?.hasEnrolledFingerprints() ?: false)
```

Comment:
```suggestion
        get() = fingerprintManager != null && fingerprintManager!!.isHardwareDetected && fingerprintManager!!.hasEnrolledFingerprints()
```

How about checking for null first and then asserting that the fingerprintManager is not null? Would clean up code imo

Reply:
Thanks for the recommendation @beatbrot !! We avoid force-unwraps `!!` unless _absolutely_ necessary, as this code is executed sequentially and the `fingerprintManager` may become null (it's a `var`) after the null-check has happened.